lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -134,8 +134,9 @@ def add_customer_data(post, options)
         post[:user][:email] = options[:email]
         post[:user][:ip_address] = options[:ip] if options[:ip]
         post[:user][:fiscal_number] = options[:fiscal_number] if options[:fiscal_number]
-        post[:user][:phone] = options[:phone] || (options[:billing_address][:phone] if options[:billing_address] &&
-          options[:billing_address][:phone])
+        if(phone = options[:phone] || (options[:billing_address][:phone]))
```

Comment:
RuboCop won't currently flag this due to a disabled check, so I'll approve despite it, but the parens are unnecessary.